ui/common/src/socket.ts

```diff
@@ -289,57 +283,65 @@ export default class StrongSocket {
     const ws = this.ws;
     if (ws) {
       this.debug('Disconnect');
-      this.autoReconnect = false;
       ws.onerror = ws.onclose = ws.onopen = ws.onmessage = () => {};
       ws.close();
     }
   };
 
-  onError = (e: unknown): void => {
+  private onError = (e: unknown): void => {
+    if (this.heartbeat.wasSuspended) return;
```

Comment:
we ignore errors when they are the expected consequence of task queue suspension